feature/platform/tests/power_admin_down_up_test/metadata.textproto

```diff
@@ -5,3 +5,17 @@ uuid: "7e1f1570-a141-4758-8aa5-228c932e3bee"
 plan_id: "FP-1.1"
 description: "Power admin DOWN/UP Test"
 testbed: TESTBED_DUT_ATE_2LINKS
+platform_exceptions: {
+  platform: {
+    vendor: CISCO
+    hardware_model: "CISCO-8808"
```

Comment:
hardware_model field is deprecated, and replaced by hardware_model_regex.

However, in this case, if this deviation applies to all CISCO models, you can remove the hardware_model.* field completly. 

So your platform_exceptions will look like this:

```
platform_exceptions: {
  platform: {
    vendor: CISCO
  }
  deviations: {
    platform_power_enable_wait: 5
    platform_power_down_up_fabric_skip: true
    platform_power_down_up_controller_card_skip: true
  }
}
```